lib/routes/account.js

```diff
@@ -959,10 +980,13 @@ module.exports = function (
         }
 
         function createResponse() {
-          return {
-             email: sessionToken.email,
-             verified: sessionToken.emailVerified
-           }
+          return db.sessionTokenVerified(sessionToken.id)
+          .then(function(token) {
+            return {
+              email: token.email,
+              verified: !!(token.emailVerified && token.tokenVerified)
```

Comment:
Is it worth adding new subfields `accountVerified` and `tokenVerified` to allow in-the-know clients to tell the difference?